packages/devextreme/js/ui/rating.d.ts

```diff
@@ -0,0 +1,186 @@
+import {
+    EventInfo,
+    NativeEventInfo,
+    InitializedEventInfo,
+    ChangedOptionInfo,
+} from '../events/index';
+
+import Editor, {
+    EditorOptions,
+    ValueChangedInfo,
+} from './editor/editor';
+
+/** @public */
+export type RatingSelectionMode = 'continuous' | 'single';
+
+/**
+ * @docid _ui_rating_ContentReadyEvent
+ * @public
+ * @type object
+ * @inherits EventInfo
+ */
+export type ContentReadyEvent = EventInfo<dxRating>;
+
+/**
+ * @docid _ui_rating_DisposingEvent
+ * @public
+ * @type object
+ * @inherits EventInfo
+ */
+export type DisposingEvent = EventInfo<dxRating>;
+
+/**
+ * @docid _ui_rating_InitializedEvent
+ * @public
+ * @type object
+ * @inherits InitializedEventInfo
+ */
+export type InitializedEvent = InitializedEventInfo<dxRating>;
+
+/**
+ * @docid _ui_rating_OptionChangedEvent
+ * @public
+ * @type object
+ * @inherits EventInfo,ChangedOptionInfo
+ */
+export type OptionChangedEvent = EventInfo<dxRating> & ChangedOptionInfo;
+
+/**
+ * @docid _ui_rating_ValueChangedEvent
+ * @public
+ * @type object
+ * @inherits NativeEventInfo,ValueChangedInfo
+ */
+export type ValueChangedEvent = NativeEventInfo<dxRating, KeyboardEvent | MouseEvent | PointerEvent | TouchEvent | Event> & ValueChangedInfo;
```

Comment:
I think that we shouldn't use Editor.ValueChangedInfo type because it includes value and previous values properties that have object type. In our case it should be number. Perhaps, that you copy this from Slider, seems that he has the same inaccurancy
https://js.devexpress.com/jQuery/Documentation/ApiReference/UI_Components/dxSlider/Configuration/#onValueChanged